src/workflows-app/utils/app-utils.js

```diff
@@ -7,6 +7,9 @@ import { AppProxyUrlStatus, getUser, workflowsAppStore } from 'src/libs/state';
 import * as Utils from 'src/libs/utils';
 import { cloudProviderTypes } from 'src/libs/workspace-utils';
 
+export const getCromwellUnsupportedMessage = () =>
```

Comment:
nit: seems like this should be a static const string and the call site can wrap it in the function if needed. Not super opinionated about this though.